src/calculateRentalCost.js

```diff
@@ -1,10 +1,18 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_ONE_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const SHORT__TERM = 3;
+  let totalPrice = PRICE_ONE_DAY * days;
+
+  if (days >= LONG_TERM) {
+    totalPrice -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT__TERM) {
+    totalPrice -= SHORT_TERM_DISCOUNT;
+  }
```

Comment:
The use of `else if` after an `if` with a `return` statement is unnecessary and can be simplified. Since you're returning inside the first `if` block, you can omit the `else` and just use a second `if` statement for the next condition. This will make the code cleaner and easier to read.